### DISCUSSION TITLE
Show multiple line on each item on the cue sheet

### DISCUSSION THREAD

**Discussion by davidtamhkcd** [cpvalente/ontime#1214] [Discussion]
On the cue sheet, Now only can show txt on single line. If able to show multiple line, the detail can show more clearly. Also if can print out the cue sheet, it is also useful

**Comment by cpvalente** [comment]
> On the cue sheet, Now only can show txt on single line. If able to show multiple line, the detail can show more clearly. Also if can print out the cue sheet, it is also useful

Hi, thank you for taking the time to make the proposal.

The custom fields are multi line and should show as so in the cuesheet. Can you please confirm that for me?

As far as printing, we have discussed this a few times. The suggested workflow is that you export the data as CSV and print from excel (or your spreadsheet software or choice). We have no current plans to build in printing into Ontime

Hope this helps

**Reply by davidtamhkcd** [reply]
Multiple line can show in a very long single line in the cue sheet, but only can show on the first maybe 20-30 characters, the rest cannot show

**Reply by cpvalente** [reply]
> Multiple line can show in a very long single line in the cue sheet, but only can show on the first maybe 20-30 characters, the rest cannot show

Thank you for this.

Can we please clarify which fields are we talking about?

The custom fields and event note are multi line. You can click on the input and add lines to confirm

All the columns have user definable width so you can just expand them as you want 

Is this not the case for you?
Which browser and OS are you on?
